test/IDE/complete_operators.swift

```diff
@@ -259,9 +260,20 @@ func testInfix15<T: P where T.T == S2>() {
 func testInfix16<T: P where T.T == S2>() {
   T.foo#^INFIX_16^#
 }
+
 func testInfix17(x: Void) {
   x#^INFIX_17^#
 }
+
+// VOID_OPERATORS: Begin completions
```

Comment:
Oh, I see how this is related to the change. `()` is same as `Void`, and therefore we get all the equality/comparison ops on the value of type `Void`. This might have other consequences...